lista-de-exercicios-java/src/main/java/org/example/exercicio03/Jokenpo.java

```diff
@@ -0,0 +1,29 @@
+package org.example.exercicio03;
+
+public class Jokenpo {
+    private JogadorBase jogador1;
+    private JogadorBase jogador2;
+
+    public Jokenpo(JogadorBase jogador1, JogadorBase jogador2) {
+        this.jogador1 = jogador1;
+        this.jogador2 = jogador2;
+    }
+
+    public String jogar() {
+        Escolhas jogador1 = this.jogador1.getEscolha();
```

Comment:
o nome destas duas variavéis poderiam deixar mais claro que se refere a jogada de um jogador bem como não repetir o mesmo nome utilizado nas linhas 4 e 5